src/ngraph/ngraph_emitter.cc

```diff
@@ -601,6 +601,33 @@ void Emitter::CreateBinaryOps() {
     return cast_result(CreateAutoBroadcast<ngraph::op::LessEq>(node),
                        getType(node->dtype_));
   };
+  ngraph_op_funcs_["SequenceMask"] = [this](const NodePtr& node) {
+    auto data = op_map_[node->inputs_[0]];
+
+    // if sequence lengths specified
+    if (node->inputs_.size() > 1) {
```

Comment:
can we also check for `use_sequence_length` , though it should be true if size > 1.

Reply:
ok